core/templates/css/oppia.css

```diff
@@ -191,7 +191,7 @@ why they need to be overriden.  */
 .md-button.oppia-learner-got-it-button,
 .md-button.oppia-learner-hint-solution-button {
   background: #0D48A1;
-  color: #ffffff;
+  color: #FFF;
```

Comment:
I think we avoid having capitalization for tags, class-names, colors etc ([ref](https://google.github.io/styleguide/htmlcssguide.html#Capitalization))

Reply:
Should we than add it as a rule into the stylelint?

Reply:
Added to stylelint and fixed it everywhere.

Reply:
Oh that's great, thanks for doing it!